concepticondata/conceptlists.tsv

```diff
@@ -355,4 +355,5 @@ List-2020-1365	List, Johann-Mattis	2020		1365	annotated	English	German	https://c
 Gerardi-2021-414	Gerardi, Fabrício and Stanislav Reichert and Coelho Aragon, Carolina	2021		414	basic	English,Portuguese	Tupían languages	https://tuled.org/parameters	Gerardi2021		This list was compiled for a large lexical database of Tupían languages and beyond. The study we provide as reference is not identical with the concept list which we present here, as it belongs to a later version that was consistently mapped to the Concepticon. As a result, the number of items differs from the number mentioned in the study presenting the database.	1-19	
-Anonby-2018-1500	Anonby, Erik and Asadi, Ashraf and Nikravan, Pegah	2018		1500	areal, questionnaire	English, Persian	Native languages of Iran	https://carleton.ca/iran/questionnaires/	Anonby2018		This comprehensive list is used as a field work questionnaire for ALI, the Atlas of the Languages of Iran project. It includes basic as well as areally specific terms and groups them into 12 semantic categories plus multiple subcategories. Many concepts are based on [Snider-2004-1700](:ref:Snider-2004-1700).		
\ No newline at end of file
+Anonby-2018-1500	Anonby, Erik and Asadi, Ashraf and Nikravan, Pegah	2018		1500	areal, questionnaire	English, Persian	Native languages of Iran	https://carleton.ca/iran/questionnaires/	Anonby2018		This comprehensive list is used as a field work questionnaire for ALI, the Atlas of the Languages of Iran project. It includes basic as well as areally specific terms and groups them into 12 semantic categories plus multiple subcategories. Many concepts are based on [Snider-2004-1700](:ref:Snider-2004-1700).		
+Duong-2020-100	Thu Hang, Duong and Thu Quynh, Nguyen and Van Loi, Nguyen	2020		100	basic, questionnaire	English	La Chí languages in Vietnam	https://evols.library.manoa.hawaii.edu/bitstream/10524/52466/1/JSEALS_Special_Publication_6_Anthropologyoflanguage.pdf	Duong2020		The current list was used to investigate the phonetic, lexical, and grammatical characteristics of the La Chí language in Bản Díu commune, in comparison with the language of La Chí in ản Máy and Bản Phùng communes in Vietnam. The concepts are claimed to be generally based on the 100-item list of Swadesh (presumably, the [one that dates back to 1955](:ref:Swadesh-1955-100)). The data for the analysis come from the materials, collected by Nguyễn Văn Lợi, Nguyễn Thu Quỳnh and Dương Thu Hằng during their field trips in the respective regions.	136-138	
```

Comment:
I propose to slightly modify the text: 

> The current list was use to investigate some basic characteristics of the La Chí language spoken in Bản Díu, comparing it with La Chí varieties in  Bản Máy and Bản Phùng

This is less similar to the author's text, and it also does not follow the omission of articles (as we find it in the original).

Reply:
> The data for the analysis come from the materials collected by Nguyễn Văn Lợi, Nguyễn Thu Quỳnh and Dương Thu Hằng during their field trips in the respective regions.

[ no comma here ]

Reply:
Achh, my passion for commas  :) Thanks for the tips!